src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -369,3 +378,69 @@ def _cc_receive(
         except Exception:
             log.exception(f"encountered error while receiving message via {self}")
             return {"msg": None}
+
+    def _merge_trc(self) -> None:
+        """Merge multiple trc files in one."""
+        list_of_traces = []
+
+        if isinstance(self.trace_path, str):
+            self.trace_path = Path(self.trace_path)
+
+        # Get all trc files in trace path
+        list_of_all_traces = list(self.trace_path.glob("*.trc"))
+
+        log.internal_debug(f"merging following trace files: {list_of_all_traces}")
+        # Select a number of the lastest trace files corresponding to the number of traces created.
+        for _ in range(self.trc_count):
+            try:
+                latest_trace = max(list_of_all_traces, key=os.path.getctime)
+                list_of_all_traces.pop(list_of_all_traces.index(latest_trace))
+                list_of_traces.append(latest_trace)
+            except ValueError:
```

Comment:
It's always a bad idea to mute an exception

Reply:
The code has been reworked to be more logical and understandable, this has been removed